front/app/index.html

```diff
@@ -90,13 +90,25 @@
     <script src="bower_components/angular-route/angular-route.js"></script>
     <script src="bower_components/angular-truncate/src/truncate.js"></script>
     <script src="bower_components/angular-ui-utils/ui-utils.js"></script>
-    <script src="bower_components/Chart.js/Chart.min.js"></script>
+    <script src="bower_components/Chart.js/Chart.js"></script>
     <script src="bower_components/ng-chartjs/dist/angular-chartjs.min.js"></script>
     <script src="bower_components/moment/moment.js"></script>
     <script src="bower_components/angular-moment/angular-moment.js"></script>
     <script src="bower_components/jquery-waypoints/waypoints.js"></script>
     <script src="bower_components/angulartics/src/angulartics.js"></script>
+    <script src="bower_components/angulartics/src/angulartics-adobe.js"></script>
```

Comment:
I'm guessing some of these were added due to an angular version update. Can take out if needed.